config/vufind/Folio.ini

```diff
@@ -101,6 +101,12 @@ extraHoldFields = requiredByDate:pickUpLocation
 ; "Recall" if a recall is preferred.
 ;default_request = Hold
 
+; Uncomment to hide the place hold/recall/page link when an item is in the configured
+; list of locations. Ideally, this would not be needed and FOLIO would include an 
+; indicator of whether or not an item is holdable with item details.
+; See https://issues.folio.org/browse/UXPROD-2422. A colon delimited list of strings.
+;excludeHoldLocations = "24 Hour Reserve":"Reference Collection"
```

Comment:
Rather than using a delimited string, it might be better to simply make this an array in the .ini file to begin with:

<pre>
;excludeHoldLocations[] = "24 Hour Reserve"
;excludeHoldLocations[] = "Reference Collection"
</pre>

... then you're less vulnerable to the quirks of .ini parsing, or to the possibility of the delimiter character being in the location name. (The delimited string approach is great for numeric IDs, but not so great for this kind of situation).

If you go this route, then you can simply skip the explode and do the in_array check directly against the configuration.

Reply:
...in fact, the whole isHoldable() function could be greatly simplified, to something like this:

<pre>
return !in_array(
    $locationName,
    (array)($this->config['Holds']['excludeHoldLocations'] ?? [])
);
</pre>

(This just defaults the setting to an empty array, and ensures that it is an array... that should support all edge cases).